src/pages/AddPost/index.jsx

```diff
@@ -10,12 +10,12 @@ export default function AddPost() {
   const [foodTags, setFoodTags] = React.useState(null);
   const [howManyPeople, setHowManyPeople] = React.useState(null);
   const [difficulty, setDifficulty] = React.useState('Easy');
-  const [ingredients, setIngredients] = React.useState(null);
-  const [howToPrepareSteps, setHowToPrepareSteps] = React.useState(null);
+  const [time, setTime] = React.useState(null);
+  const [ingredients, setIngredients] = React.useState({});
+  const [howToPrepareSteps, setHowToPrepareSteps] = React.useState([]);
   //The newPostCheckBox holds the food options and their state, if the option doesnt 
```

Comment:
i don't see howManyPeople being used here, you set it, but don't use the state. this might cause unexpected bugs in the future, same goes for time and foodTags & difficulty